### PR TITLE
Fix tabbed content bug in default js (#2538)

### LINKED ISSUE #2538: Tabbed content switch doesn't work if Tab name contains not plain text
**DocFX Version Used**:  2.31

**Template used**: default

**Steps to Reproduce**:

1.  Add a markdown document with text like this in the project
```
# [**Tab Name**](#tab/tab-id-1)
Content1 ...
# [**Tab Name**](#tab/tab-id-2)
Content2 ...
---
```

> Note that "Tab Name" is bold

2. Build the docfx project
3. Open result document in a browser

**Expected Behavior**: Page with tabbed content that can be switched

**Actual Behavior**: Tabbed content can't be switched

Click event handling doesn't work if you click something other than HTMLAnchorElement

### PR DESCRIPTION
**Pull request by vicancy**
Fix #2538

### CODE REVIEW

File: src/docfx.website.themes/default/styles/docfx.js [hunk 1]
```diff
@@ -37,6 +37,8 @@ $(function () {
     renderTabs();
   }
 
+  // window.addEventListener('content-update', contentUpdate);
```
Comment (superyyrrzz) [inline src/docfx.website.themes/default/styles/docfx.js]: > // window.addEventListener('content-update', contentUpdate); [](start = 1, length = 61)

remove?

File: src/docfx.website.themes/default/styles/docfx.js [hunk 2]
```diff
@@ -719,227 +721,269 @@ $(function () {
...
+            return true;
+          }
+        }
+      }
+      return false;
+    }
+
+    function notifyContentUpdated() {
+      // window.dispatchEvent(new CustomEvent('content-update'));
+    }
```
Comment (superyyrrzz) [inline src/docfx.website.themes/default/styles/docfx.js]: remove?

File: src/docfx.website.themes/default/styles/docfx.js (hunk 1 above)
Comment (vicancy) [reply src/docfx.website.themes/default/styles/docfx.js]: Keep and add a comment

File: src/docfx.website.themes/default/styles/docfx.js (hunk 2 above)
Comment (vicancy) [reply src/docfx.website.themes/default/styles/docfx.js]: Keep and add a comment

**Pull request merged**